back/engines/commercial/admin_api/app/controllers/admin_api/tenants_controller.rb

```diff
@@ -94,7 +94,7 @@ def set_tenant
     end
 
     def config_params
-      @config_params ||= params.require(:tenant).permit(:host, :name, :logo, :header_bg, settings: {}, style: {})
+      @config_params ||= params.require(:tenant).permit(:host, :name, :logo, settings: {}, style: {})
```

Comment:
See [comment](https://github.com/CitizenLabDotCo/citizenlab/pull/2230/files#r960366946) on deleting this from `tenants`.